src/main/java/org/broadinstitute/consent/http/mail/freemarker/NewDaaRequestModel.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.consent.http.mail.freemarker;
+
+import java.util.List;
+import java.util.Map;
+import org.broadinstitute.consent.http.models.Dac;
+import org.broadinstitute.consent.http.models.Dataset;
```

Comment:
These imports appear unused (I think this would have also been caught by a linter).